prometheus-exporters/ucs-exporter/templates/ucs-exporter-deployment.yaml

```diff
@@ -0,0 +1,65 @@
+kind: Deployment
+{{- if .Capabilities.APIVersions.Has "apps/v1" }}
+apiVersion: apps/v1
+{{- else }}
+apiVersion: extensions/v1beta1
+{{- end }}
+metadata:
+  name: ucs-exporter 
+  namespace: infra-monitoring
+spec:
+  selector:
+    matchLabels:
+      app: ucs-exporter 
+      type: exporter
+  template:
+    metadata:
+      labels:
+        app: ucs-exporter
+        type: exporter
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: {{ required ".Values.ucs.port_number missing" .Values.ucs.port_number | quote}}
+        prometheus.io/targets: {{ required ".Values.ucs.alerts.prometheus missing" .Values.ucs.alerts.prometheus | quote }}
+    spec:
+      containers:
+         - name: ucs-exporter
+           image: {{ required ".Values.global.registry missing" .Values.global.registry }}/{{ required ".Values.ucs.image.name missing" .Values.ucs.image.name }}:{{ required ".Values.ucs.image.tag missing" .Values.ucs.image.tag  }}
+           ports:
+             - name: metrics
+               containerPort: {{ required ".Values.ucs.port_number missing" .Values.ucs.port_number }}
+           command:
+             - ./ucs-exporter.py
+           args:
+             - -u
+             - {{ required ".Values.ucs.user missing" .Values.ucs.user }}
+             - -p
+             - {{ required ".Values.ucs.master_password missing" .Values.ucs.master_password }}
```

Comment:
please change to `.Values.global.master_password`

Reply:
Done